wandb/agent.py

```diff
@@ -35,6 +38,13 @@ def __init__(self, api, queue, sweep_id=None):
         self._sweep_id = sweep_id
         self._log = []
         self._running = True
+        self._last_report_time = None
+        self._report_interval = wandb.env.get_agent_report_interval(self.REPORT_INTERVAL)
+        self._kill_delay = wandb.env.get_agent_kill_delay(self.KILL_DELAY)
+        if self._report_interval is None:
```

Comment:
These exceptions are not ideal, and I intentionally wasn't adding cli parameters because I think this needs a more complete fix later.